crates/oxc_prettier/src/printer.rs

```diff
@@ -58,23 +61,73 @@ impl<'a> Printer<'a> {
     /// * <https://github.com/prettier/prettier/blob/0176a33db442e498fdb577784deaa77d7c9ae723/src/document/printer.js#L302>
     fn print_doc_to_string(self) -> String {
         let mut out = vec![];
-        let mut cmds: Vec<Command> = vec![Command::new(Indent::root(), Mode::Break, self.doc)];
+        let mut cmds: Vec<Command> = vec![Command::new(Indent::root(), Mode::Break, &self.doc)];
+
+        let mut pos: u16 = 0;
```

Comment:
should this be a `u16`?

Reply:
Performance should be negligible in such cases, use the shorter one until it blows up ;-)